components/calendar/src/julian.rs

```diff
@@ -253,33 +272,67 @@ impl Julian {
 
     // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/1ee51ecfaae6f856b0d7de3e36e9042100b4f424/calendar.l#L1711-L1738
     fn julian_from_fixed(date: RataDie) -> JulianDateInner {
-        let approx = quotient64((4 * date.to_i64_date()) + 1464, 1461);
-        let year = if approx <= 0 { approx - 1 } else { approx };
-        let year = match i64_to_i32(year) {
+        let approx = quotient64(4 * date.to_i64_date(), 1461) + 1;
```

Comment:
what happened to 1464, was it not necessary?

Reply:
I extracted it as the `+ 1` at the end while debugging, but I've put it back in for consistency